<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCostomer</class>
 <widget class="QWidget" name="AddCostomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;
border: 1px solid #ccc;
border-radius: 5px;
background-image: url(:/ing/AddCustImg.png);
background-repeat:no-repeat;
background-position:center;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>1164</width>
          <height>590</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1164</width>
          <height>590</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;
background-color: rgb(255, 255, 255);

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="3">
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
    background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Light shadow for depth */
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QDateEdit" name="AC_DateOfBirth">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QDateEdit {
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px 10px;
    background-color: #ffffff;
    font-size: 14px;
    color: #444;
    width: 150px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Light shadow for a floating effect */
}

QDateEdit::hover {
    border: 1px solid #bbb;
    background-color: #f7f7f7f7;
}

QDateEdit::focus {
    border: 1px solid #4CAF50;
    background-color: #e0f7e7;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2); /* Stronger shadow on focus */
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ddd;
    background-color: #f0f0f0;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
    padding-right: 5px;
}

QDateEdit::down-arrow {
    width: 12px;
    height: 12px;
}

QDateEdit::down-arrow:hover {
}

QDateEdit::up-arrow, QDateEdit::down-arrow {
    width: 12px;
    height: 12px;
}

QDateEdit::up-button, QDateEdit::down-button {
    subcontrol-origin: border;
    subcontrol-position: center right;
    width: 16px;
    height: 16px;
    background-color: transparent;
}

QDateEdit::drop-down:pressed {
    background-color: #d6d6d6;
}

QDateEdit::drop-down:focus {
    border-left: 1px solid #4CAF50;
}

QDateEdit QAbstractItemView {
    border: 1px solid #ccc;
    border-radius: 4px;
    background-color: #f7f7f7;
    selection-background-color: #4CAF50;
    selection-color: #fff;
}

</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_10">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0,0,0);
border:none;
</string>
              </property>
              <property name="text">
               <string>(MM/DD/YYYY)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;color: rgb(0,0,0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Contact No.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0,0,0);
border:none;</string>
           </property>
           <property name="text">
            <string>Customer ID</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="AC_CustomerID">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="Cancel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:none;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="AC_PassportNo">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QTextEdit" name="AC_Address">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Styling the Frame that contains the Radio Buttons */
QFrame {
    background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Light shadow for depth */
}

/* Styling the Radio Buttons */
QRadioButton {
    font-size: 14px;
    color: #444;
    spacing: 8px; /* Space between the radio button and the text */
}

/* Circle Style for Radio Buttons */
QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    border: 2px solid #aaa;
    background-color: #fff;
    margin-right: 4px;
}

/* Checked State */
QRadioButton::indicator:checked {
    border: 2px solid #4CAF50;
    background-color: #4CAF50;
}

/* Hover Effect */
QRadioButton::indicator:hover {
    border: 2px solid #888;
}

/* Focused State */
QRadioButton::indicator:focus {
    border: 2px solid #4CAF50;
}

/* Disabled State */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background-color: #eee;
}

QRadioButton::indicator:checked:disabled {
    background-color: #a5d6a7;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QRadioButton" name="AC_Female">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0,0,0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Female</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="AC_Male">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0,0,0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Male</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;color: rgb(0,0,0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Gender</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="AC_ContentNo">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="AC_LastName">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="AC_NID">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0,0,0);
border:none;</string>
           </property>
           <property name="text">
            <string>First Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0,0,0);
border:none;</string>
           </property>
           <property name="text">
            <string>Passport No.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0,0,0);
border:none;</string>
           </property>
           <property name="text">
            <string>Last Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="AC_FirstName">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="AC_Add">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:none;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;background:none;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0,0,0);</string>
           </property>
           <property name="text">
            <string>National ID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;color: rgb(0,0,0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Date OF Birth</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;background:none;
color: rgb(0,0,0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#label_11 {
background:none;
    background-color: #f7f7f7; /* Light grey background */
    color: #333333; /* Dark grey text */
    border: 1px solid #d9d9d9; /* Light grey border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Padding around the text */
    font-size: 20px; /* Font size appropriate for the label size */
    font-weight: bold; /* Bold text */
    text-align: center; /* Center the text */
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Subtle shadow effect */
    margin-bottom: 10px; /* Space below the label */
    letter-spacing: 1px; /* Spacing between letters */
    height: 30px; /* Explicit height for the label */
}
</string>
        </property>
        <property name="text">
         <string>Add Customer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>on_Cancel_clicked()</slot>
 </slots>
</ui>
